.travis.yml

```diff
@@ -12,7 +12,7 @@ addons:
                         - rpm
 install:
         - ./autogen.sh
-        - ./configure --prefix=$HOME
+        - if [ "$TRAVIS_OS_NAME" = "linux" and "$CC" = "gcc"]; then ./configure --prefix=$HOME --enable-debug; else ./configure --prefix=$HOME; fi
```

Comment:
So you mean, execute the steps as before and an additional step to build with -enable-debug in for linux/gcc?